src/components/tag/index.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import React from 'react';
+
+import { IProps } from './types';
+
+/**
+ * @brief Tag 컴포넌트
+ * @description label 필수, style 추가 수정 가능
+ * @param label 제목
+ * @param children 아이콘 같은 것 넣으면 될 것 같음
+ * @param className tailwind 요소
+ */
+
+const Tag = ({ label, children, className, ...props }: IProps) => {
+  return (
+    <div
+      className={`${className} flex h-[32px] w-[55px] items-center justify-center rounded-lg border-primaries-primary bg-primaries-primary text-text-20 hover:bg-primaries-hover active:bg-primaries-active`}
```

Comment:
h,w를 명시하지 않고 inline-block을 주면 자동으로 텍스트에 맞게 크기가 적용될 것 같습니다!
+) 피그마처럼 active일 때는 기본 색상이면 더 좋을 것 같아서 이렇게 수정해 보았는데....민수님은 어떻게 생각하시나용? 🙄

```suggestion
      className={`${className} inline-block items-center rounded-lg border-primaries-primary bg-primaries-primary px-[12px] py-[5px] text-text-20 hover:bg-primaries-hover`}

```